src/source.cc

```diff
@@ -2120,7 +2121,25 @@ bool Source::View::on_button_press_event(GdkEventButton *event) {
     get_buffer()->select_range(start, end);
     return true;
   }
-  
+  if((event->type == GDK_BUTTON_PRESS) && (event->button == 3)){
+    auto menu=Menu::get().edit_menu;
```

Comment:
I'm not convinced that two popup menus are the way to go... It could get confusing and in what situations are the menu items different?

I agree that a separate schema for right click is better.